app/controllers/web/admin/news_controller.rb

```diff
@@ -14,16 +14,27 @@ def show
   def create
     @news = News.new
     @news_form = NewsForm.new @news
+
+    #fix because sessions work wrong
+    params[:news][:user_id] = current_user.id if current_user.present?
+
     @news_form.submit params[:news]
     if @news_form.save
       redirect_to admin_news_index_path
     else
-      redirect_to action: :new
+      flash[:errors] = @news_form.errors
+      flash[:model_name_class] = 'News'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.